opsdroid/const.py

```diff
@@ -29,6 +29,8 @@
 RASANLU_DEFAULT_URL = "http://localhost:5000"
 RASANLU_DEFAULT_PROJECT = "opsdroid"
 
+SNIPSNLU_DEFAULT_DIR = "/home/snipsmodel"
```

Comment:
This isn't really a suitable location for this. User's may not want things dropped in their home directory like this, and it will not work on Windows.

I recommend that we place them within the `DEFAULT_ROOT_PATH` directory which uses `appdirs` to identify a suitable location for the current operating system.

I would suggest that we add a new constant called `DEFAULT_MODELS_PATH` as a central place to store local NLU models, and then place a `snips` directory within that.

```suggestion
DEFAULT_MODELS_PATH = os.path.join(DEFAULT_ROOT_PATH, 'models')
SNIPSNLU_DEFAULT_DIR = os.path.join(DEFAULT_MODELS_PATH, 'snips')
```